src/popup/components/Modals/EnableSecureLogin.vue

```diff
@@ -0,0 +1,141 @@
+      <IconBoxed
+        :icon="FingerprintIcon"
+        small-icon
+      />
+    </div>
+    <div class="info">
+      <h3 class="title">
+        {{ $t('pages.titles.secureLogin') }}
+      </h3>
+
+      <div class="text">
+        <span>{{ isAvailable
+          ? $t('pages.secureLogin.available.part1')
+          : $t('pages.secureLogin.unavailable.part1')
+        }}</span>
+        <span>{{ isAvailable
+          ? $t('pages.secureLogin.available.part2')
+          : $t('pages.secureLogin.unavailable.part2')
+        }}</span>
+      </div>
+
+      <div class="buttons">
+        <BtnMain
+          variant="primary"
+          extend
+          :text="isAvailable
+            ? $t('pages.secureLogin.enableSecureLogin')
+            : $t('pages.secureLogin.enableDeviceSecureLogin')
+          "
+          @click="onEnable"
+        />
+        <BtnMain
+          variant="muted"
+          extend
+          :text="$t('common.notNow')"
+          @click="resolve"
+        />
+      </div>
+    </div>
+  </Modal>
+</template>
+
+<script lang="ts">
+import { defineComponent, PropType } from 'vue';
+import { NativeSettings, AndroidSettings, IOSSettings } from 'capacitor-native-settings';
+import type { RejectCallback, ResolveCallback } from '@/types';
+import { useBiometricAuth, useUi } from '@/composables';
+
+import Modal from '../Modal.vue';
+import BtnMain from '../buttons/BtnMain.vue';
+import IconBoxed from '../IconBoxed.vue';
+
+import FingerprintIcon from '../../../icons/fingerprint.svg?vue-component';
+
+export default defineComponent({
+  components: {
+    Modal,
+    BtnMain,
+    IconBoxed,
+  },
+  props: {
+    resolve: { type: Function as PropType<ResolveCallback>, required: true },
+    reject: { type: Function as PropType<RejectCallback>, required: true },
+  },
+  setup(props) {
+    const { isAvailable, openAuthModal } = useBiometricAuth();
+    const { setSecureLoginEnabled } = useUi();
+
+    function onEnable() {
+      if (!isAvailable.value) {
+        NativeSettings.open({
+          optionAndroid: AndroidSettings.Security,
+          optionIOS: IOSSettings.TouchIdPasscode,
+        });
+        props.resolve();
+        return;
+      }
+
+      setSecureLoginEnabled(true);
+      openAuthModal();
+      props.resolve();
+    }
+
+    return {
+      FingerprintIcon,
+      isAvailable,
+      onEnable,
+    };
+  },
+});
+</script>
+
+<style lang="scss" scoped>
+@use '../../../styles/variables' as *;
+@use '../../../styles/mixins';
+@use '../../../styles/typography';
+
+.enable-secure-login {
+  background-color: $color-bg-app;
+
+  .icon-wrapper {
+    margin-top: 44px;
+    opacity: 0.5;
+  }
+
+  .info {
+    .title {
+      color: $color-white;
+      padding:16px 0;
+
+      @extend %face-sans-18-medium;
+    }
+
+    .text {
+      color: rgba($color-white, 0.85);
+      line-height: 20px;
+      gap: 10px;
+
+      @extend %face-sans-14-light;
+
+      @include mixins.flex(center, center, column);
```

Comment:
I strongly advise against using this mixin as it gives more harm than real value.